Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs

```diff
@@ -25,16 +27,19 @@ public sealed class OrchardApplicationFactory<TStartup> : WebApplicationFactory<
     private readonly Action<IConfigurationBuilder> _configureHost;
     private readonly Action<IWebHostBuilder> _configuration;
     private readonly Action<ConfigurationManager, OrchardCoreBuilder> _configureOrchard;
-    private readonly List<IStore> _createdStores = new();
+    private readonly ConcurrentBag<IStore> _createdStores = new();
+    private readonly ICounterDataCollector _counterDataCollector;
```

Comment:
Keep the order of private fields and ctor assignments the same as the ctor parameters.